base/server/python/pki/server/__init__.py

```diff
@@ -73,6 +73,51 @@ def instances(cls):
 
         return instances
 
+    @staticmethod
+    def load_audit_events(filename):
+
+        logger.info('Loading %s', filename)
+
+        with open(filename) as f:
+            lines = f.read().splitlines()
+
+        events = {}
+
+        event_pattern = re.compile(r'# Event: (\S+)')
+        subsystems_pattern = re.compile(r'# Applicable subsystems: (.*)')
```

Comment:
Since you are parsing based on the existence of "# Event:" and "# Applicable subsystems:", I suggest you put a "WARNING" at top of the audit-events.properties about the significance of these two comment lines and the exact syntax that events need to conform to.